### DISCUSSION TITLE
Is it possible to have `postUpgradeTasks` in `packageRules`?

### DISCUSSION THREAD

**Discussion by hisener** [renovatebot/renovate#8517] [Request Help] [answered]
Hey, we were thinking about fixing breaking changes for our internal libraries as part of Renovate upgrade. Is it possible to have a post-upgrade task for a specific dependency? Something like:

```
{
  "packageRules": [
    {
      "matchPackagePatterns": [
        "internal-library"
      ],
      "postUpgradeTasks": {
        "command": "docker run ..."
      }
    }
  ]
}
```

**Comment by rarkins** [comment, marked as answer]
Yes, I think that should be possible. `postUpgradeTasks` seem to be attached to each *upgrade*

**Reply by hisener** [reply]
👌  I will give it a try later.

**Reply by MartinodF** [reply]
For posterity's sake, since this is the first hit on Google for `postUpgradeTasks packageRules`: yes, it is possible and works as expected!

**Reply by felipecrs** [reply]
Even though the JSON schema does not list it as a supported key.
